tasks/bundle-js.js

```diff
@@ -108,8 +108,8 @@ async function bundleJS(/** @type {JSEntry} */entry, {production}) {
                 cacheRoot: production ? null : `${fs.realpathSync(os.tmpdir())}/darkreader_typescript_cache`,
             }),
             rollupPluginReplace({
-                '__DEBUG__': production ? 'false' : 'true',
-                '__PORT__': production ? '-1' : String(PORT),
+                '__WATCH__': production ? 'true' : 'false',
```

Comment:
Both `__DEBUG__` and `__WATCH__` should be there. Search for `production` all over the code and replace it with `debug` and add `watch` where necessary.